### ISSUE TITLE
[Triggers]  On score < 50 propagate Visits to dynamodb

### ISSUE THREAD

**Issue by simontegg** [Ecosify/COVID-Community-Health-App#31]
depends on #29 and #30 

Visits on day of report + the 3 days prior

Initial phase will just have these update to the dynamo via a regular mutation

**Comment by simontegg** [comment]
closing in favour of #50
